packages/BetterPhpDocParser/PhpDocParser/BetterPhpDocParser.php

```diff
@@ -40,7 +40,25 @@ public function __construct(
         private readonly iterable $phpDocNodeDecorators,
         private readonly PrivatesAccessor $privatesAccessor = new PrivatesAccessor(),
     ) {
-        parent::__construct($typeParser, $constExprParser);
+        parent::__construct(
+            // TypeParser
+            $typeParser,
+            // ConstExprParser
+            $constExprParser,
+            // requireWhitespaceBeforeDescription
+            false,
+            // preserveTypeAliasesWithInvalidTypes
+            false,
+            // usedAttributes
+            ['lines' => true, 'indexes' => true],
```

Comment:
this is based on @ondrejmirtes suggestion at https://github.com/rectorphp/rector/issues/7959#issuecomment-1646571229

Reply:
Make sure to pass the same array also to other classes that use it - ConstExprParser, TypeParser, PhpDocParser.